gradle/docker.gradle

```diff
@@ -105,81 +73,93 @@ task cleanSimpleStack(type: Delete, dependsOn: stopSimpleStack) {
   doLast {
-    def envTemplatePath = "${radarStackPath}/etc/env.template"
-    def smtpTemplatePath = "${radarStackPath}/etc/smtp.env.template"
-    def streamTemplatePath = "${radarStackPath}/etc/radar.yml.template"
-
     def volumes = file("${radarStackPath}/volumes")
     volumes.mkdir()
 
-    def fileEnv = file(envTemplatePath + 'tmp')
-    fileEnv << file(envTemplatePath).text
-    fileEnv.text = fileEnv.text.replace('/usr/local/var/lib/docker', volumes.getAbsolutePath())
-    fileEnv.text = fileEnv.text.replace('HOTSTORAGE_USERNAME=<mongodb-user>', 'HOTSTORAGE_USERNAME=restapi')
-    fileEnv.text = fileEnv.text.replace('HOTSTORAGE_PASSWORD=XXXXXXXX', 'HOTSTORAGE_PASSWORD=radarcns')
-    fileEnv.text = fileEnv.text.replace('HOTSTORAGE_NAME=<mongodb-database>', 'HOTSTORAGE_NAME=hotstorage')
-    fileEnv.renameTo("${radarStackPath}/.env")
-
-    def fileSmtp = file(smtpTemplatePath + 'tmp')
-    fileSmtp << file(smtpTemplatePath).text
-    fileSmtp.renameTo("${radarStackPath}/etc/smtp.env")
-
-    def fileStream = file(streamTemplatePath + 'tmp')
-    fileStream << file(streamTemplatePath).text
-    fileStream.renameTo("${radarStackPath}/etc/radar.yml")
-    
-    def mpYml = file("${simpleStackPath}/resources/mp_info.yml")
-    file("${radarStackPath}/etc/rest-api/mp_info.yml") << mpYml.text
-
     def installScript = file("${radarStackPath}/install-radar-stack.sh")
     installScript.text = installScript.text.replace(' docker-compose ', " ${dockerCompose} ")
 
     def dockerComposeYml = file("${radarStackPath}/${radarDockerCompose}")
-    file("${radarStackPath}/${radarDockerComposeOriginal}") << dockerComposeYml.text
-    def dockerComposeTmp = file("${radarStackPath}/.${radarDockerCompose}.tmp")
-    dockerComposeTmp << dockerComposeYml.text.replaceAll('radarcns/radar-restapi:.*', 'radarcns/radar-restapi:dev')
-
-    // Add wiremock as a dependency to Rest Api
-    def dockerComposeText = dockerComposeTmp.text.replaceAll('  rest-api:\n' +
-            '    image: radarcns/radar-restapi:dev\n' +
-            '    networks:\n' +
-            '      - hotstorage\n' +
-            '      - api\n' +
-            '    depends_on:\n' +
-            '      - hotstorage\n' +
-            '    restart: always', '  rest-api:\n' +
-            '    image: radarcns/radar-restapi:dev\n' +
-            '    networks:\n' +
-            '      - hotstorage\n' +
-            '      - api\n' +
-            '    depends_on:\n' +
-            '      - hotstorage\n' +
-            '      - wiremock\n' +
-            '    restart: always')
-    dockerComposeTmp.createNewFile()
-    dockerComposeTmp.write(dockerComposeText)
-    // Add wiremock to docker-compose to start up with the stack
-    dockerComposeTmp.append("\n\n  wiremock:\n" +
+    dockerComposeYml.text = dockerComposeYml.text.replaceAll('radarcns/radar-restapi:.*', 'radarcns/radar-restapi:dev')
+    dockerComposeYml.text += "\n\n  wiremock:\n" +
```

Comment:
If removed wiremock then i guess this block of code is not required.

Reply:
True. The wiremock is also used in the e2e test, which is not removed yet. I was working on another branch to fix the e2e test. This will be removed when we fully dockerize e2e test too. In this branch i am not using e2e tests

Reply:
Ahh okay. got it! 👍